examples/src/main/python/mllib/naive_bayes_example.py

```diff
@@ -38,8 +45,12 @@ def parseLine(line):
 
     sc = SparkContext(appName="PythonNaiveBayesExample")
 
+    WORK_DIR = './'
```

Comment:
This is not necessary. There is a tradeoff between code simplicity and robustness. The main purpose of this example code is to be included in the user guide. See the source code of https://github.com/apache/spark/blob/master/docs/mllib-naive-bayes.md. So simplicity is more important.